packages/lambdas/src/sqs-to-converter.ts

```diff
@@ -181,7 +182,8 @@ export const handler = Sentry.AWSLambda.wrapHandler(async (event: SQSEvent) => {
         await ossApi.notifyApi({ ...lambdaParams, status: "failed" }, log);
         return;
       }
-      const payloadClean = cleanUpPayload(payloadRaw);
+      const payloadNoB64 = removeBase64PdfEntries(payloadRaw);
+      const payloadClean = cleanUpPayload(payloadNoB64);
```

Comment:
Can we call `removeBase64PdfEntries` from `cleanUpPayload`? That's what the clean up function does, and it already calls other functions for specific cleaning ups.

Reply:
No I think its important to be separate because v2 of this will be extracting the b64, creating docRefs, saving to FHIR, and storing on s3.